CUMULUS-2293: RDS Data migration & upgrade docs (#2195)
nasa/cumulus: lambdas/data-migration2/README.md

# Data Migration 2 Lambda

The data-migration2 lambda is the second of two Lambdas to be run in order to migrate existing DynamoDB data to PostgreSQL.

### Running the API

> This second Lambda in the data migration process can be by invoking an async operation using the provided `${PREFIX}-postgres-migration-async-operation` lambda and invoking from the AWS console or command line.

Author: ```suggestion
This second Lambda in the data migration process can berun  by invoking the provided `${PREFIX}-postgres-migration-async-operation` Lambda and invoking from the AWS console or command line.
```

Reviewer: [`c80c01f` (#2195)](https://github.com/nasa/cumulus/pull/2195/commits/c80c01f06d47dda7afbf7ed0e6d040f5a1da1870)

# Data Migration 2 Lambda

The data-migration2 lambda is the second of two Lambdas to be run in order to migrate existing DynamoDB data to PostgreSQL.

### Running the API
This second Lambda in the data migration process can be run in several ways:

1. In an ECS container using the Cumulus ECS Service
2. Directly from the AWS console / command line

> 3. By invoking an async operation using the provided `${PREFIX}-postgres-migration-async-operation` lambda and invoking from the AWS console or command line.

Author: Option 3 is just a repeat of option 2 with more detail. And option 1 is no longer valid. We should just make it clear that there is only one way to run this migration.

> Running the Lambda within an ECS container is a more common approach as it is likely this execution will take some time. See `/example/data-migration-ecs` for an example container configuration.

Author: This is inaccurate now. That example doesn't exist. People just run this Lambda and it transparently handles starting an async operation for them

> Alternatively, you can invoke the `${PREFIX}-postgres-migration-async-operation` lambda to start an async operation to invoke the data-migration2 lambda. For more information, see [README](../../lambdas/postgres-migration-async-operation/README.md)

Author: It's not really an alternative, that's the only way we should document it as supported given the need to use ECS for the sake of memory usage and runtime. No one could/should really just run this Lambda directly as they would at least hit the 15 minute Lambda timeout
